JSONFormat.go

```diff
@@ -39,7 +45,7 @@ func (j jsonEntry) MarshalJSON() ([]byte, error) {
 		return nil, ErrNoMessage
 	}
 
-	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, j.Message))
+	_, err := buf.WriteString(fmt.Sprintf("{\"%s\":\"%s\", \"%s\":\"%s\", \"%s\":\"%s\"", j.levelKey, j.Level, j.eventKey, j.Event, j.messageKey, escapeSpecialChars(j.Message)))
```

Comment:
In this gist there are the two version of `marshalJson`: https://gist.github.com/markvincze/014bd9ba9101f8d3f4c1dfbb85c737a8
So as a developer, you find the first one better, and that's the one you'd prefer writing and maintaining?
And another question: would you do the same thing if you had to *parse* json?

>Go is low level, don't need anything but the standard library :P

`json.Marshal` is part of the standard library, right?
And I think a language being low level is not a real argument against using a mature, reliable library instead of writing low-level code. (For example I've written quite a lot of C++, which is even more low level, but I've never written XML or Json serialization manually.)

>Personally, too high level and you go into "magic" behavior and I hate magic.
>...
>For me, the less magic, the dumber the code, the better.

But would you consider Json serialization high level magic? I mean what can be the problem with it, that it contains bug, or that it's slow? I would much rather trust the built in, or a well-known open-source library than the custom code we're writing.

And this argument could be applied to many other things, for example we could say let's not use a library for HTTP communication, but let's construct the HTTP message and parse the HTTP response manually. But we don't normally do that, do we?